### PR TITLE
add tablet support

### PR DESCRIPTION
**Pull request by BusyJay**

### CODE REVIEW

File: proto/raft_serverpb.proto [hunk 1]
```diff
@@ -103,6 +103,8 @@ message RegionLocalState {
     PeerState state = 1;
     metapb.Region region = 2;
     MergeState merge_state = 3;
+    // The apply index corresponding to the storage when it's initialized.
```
Comment (Connor1996) [inline proto/raft_serverpb.proto:106]: apply index？

Comment (BusyJay) [reply proto/raft_serverpb.proto:106]: Yes. tablet index is actually (max applied) index.

Comment (Connor1996) [reply proto/raft_serverpb.proto:106]: how about describing the tablet naming rule here

Comment (BusyJay) [reply proto/raft_serverpb.proto:106]: It's TiKV's implementation details, I would like to make it inside TiKV.

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by 5kbpers — APPROVED** [review]

**Pull request merged**